### PR TITLE
Simplify product tests launching

### PR DESCRIPTION
**Pull request by wendigo**
Follow-up to https://github.com/prestosql/presto/pull/4702

Make `product-tests-suite-*.sh` simpler by reducing boilerplate

### CODE REVIEW

File: presto-product-tests-launcher/bin/run-product-tests [hunk 1]
```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env bash
```
Comment (kokosing) [inline presto-product-tests-launcher/bin/run-product-tests:1]: Sorry, I don't find it simpler. Yet another layer to me makes it a bit more complex.

File: presto-product-tests-launcher/bin/run-product-tests [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+    echo "Environment name is missing (use --environment)"
+    exit 1
+fi
+
+if [ "$TEMPTO_ARGS" = "" ]; then
+    echo "Tempto run configuration is missing (use combination of -g -x -t -e)";
+    exit 1
+fi
+
+REPORTS_DIR="${BASH_SOURCE%/*}/../../presto-product-tests/target/${SUITE}-${ENVIRONMENT}"
```
Comment (kokosing) [inline presto-product-tests-launcher/bin/run-product-tests:78]: maybe we could generate report dir in java launcher code. That could use information about environment and possibly tests groups. What do you think?

Comment (wendigo) [reply presto-product-tests-launcher/bin/run-product-tests:78]: I don't know how to ensure that this will be unique

**Comment by wendigo** [comment]
@electrum thoughts?

File: presto-product-tests-launcher/bin/run-product-tests (hunk 2 above)
Comment (findepi) [reply presto-product-tests-launcher/bin/run-product-tests:78]: why not have the report name dirs such as 

```
......./test-report-${ENVIRONMENT}-yyyy-MM-dd-HH-mm-ss
```

or


```
......./test-report-${ENVIRONMENT}-uuid
```

and yes, doing it in Java would be nice

Comment (wendigo) [reply presto-product-tests-launcher/bin/run-product-tests:78]: @findepi I guess it makes sense if we put it under directory which gets cleaned up i.e. presto-product-tests/target or something like that

**Pull request closed without merging**